tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -345,27 +367,18 @@ async def test_nft_wallet_rpc_creation_and_list(two_wallet_nodes: Any, trusted:
     await time_out_assert_not_none(15, full_node_api.full_node.mempool_manager.get_spendbundle, sb.name())
     for i in range(1, num_blocks):
         await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
-    time_left = 5.0
-    while time_left > 0:
-        coins_response = await api_0.nft_get_nfts(dict(wallet_id=nft_wallet_0_id))
-        try:
-            assert isinstance(coins_response, dict)
-            assert coins_response.get("success")
-            coins = coins_response["nft_list"]
-            assert len(coins) == 2
-            uris = []
-            for coin in coins:
-                assert not coin.supports_did
-                uris.append(coin.data_uris[0])
-                assert coin.mint_height > 0
-            assert len(uris) == 2
-            assert "https://chialisp.com/img/logo.svg" in uris
-            assert bytes32.fromhex(coins[1].to_json_dict()["nft_coin_id"][2:]) in [x.name() for x in sb.additions()]
-        except AssertionError:
-            if time_left < 1:
-                raise
-        await asyncio.sleep(0.5)
-        time_left -= 0.5
+    coins_response = await wait_rpc_state_condition(
+        5, api_0.nft_get_nfts, [{"wallet_id": nft_wallet_0_id}], lambda x: x["success"] and len(x["nft_list"]) == 2
```

Comment:
Could we pass by name so we can read what each thing is here?  That would also end up with each parameter on a separate line which will be more readable as well.

Reply:
pass by name?